src/Database/Relational.php

```diff
@@ -1338,27 +1338,50 @@ public function save()
 		return $result;
 	}
 
+	/**
+	 * Filters out fields that are not actually a table column
+	 * @param	string	$table	The name of the database table
+	 * @param	array	$data	The attributes passed in to be saved to the table
+	 * @return	array
+	 **/
+	public function getTableColumnsOnly()
+	{
+		$data = array();
+		$tableColumns = $this->getStructure()->getTableColumns($this->table);
+		if (!empty($tableColumns))
+		{
+			$data = array_intersect_key($this->attributes, $tableColumns);
+		}
+		if (isset($data['shorttitle']) || isset($data['file']))
+		{
+			print_r($data);
+			exit();
```

Comment:
What's with these two lines?

Reply:
Whoops, forgot to remove that.  Should be good now.